<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="D90D3CQ1" name="com.scripts.desktopGNG.Reception_Entry" tests="1" failures="0" timestamp="25 Apr 2017 10:22:35 GMT" time="2.221" errors="1">
  <testcase name="loggin" time="2.221" classname="com.scripts.desktopGNG.Reception_Entry">
    <error type="java.lang.VerifyError" message="Expecting a stackmap frame at branch target 17
Exception Details:
  Location:
    com/generic/Utilities.waitFor(I)V @0: getstatic
  Reason:
    Expected stackmap frame at this location.
  Bytecode:
    0x0000000: b200 8d1b 85b2 0093 b600 96b8 009a a700
    0x0000010: 0d4d bb00 a059 2cb7 00a2 bfb1          
  Exception Handler Table:
    bci [0, 14] =&gt; handler: 17
">
      <![CDATA[java.lang.VerifyError: Expecting a stackmap frame at branch target 17
Exception Details:
  Location:
    com/generic/Utilities.waitFor(I)V @0: getstatic
  Reason:
    Expected stackmap frame at this location.
  Bytecode:
    0x0000000: b200 8d1b 85b2 0093 b600 96b8 009a a700
    0x0000010: 0d4d bb00 a059 2cb7 00a2 bfb1          
  Exception Handler Table:
    bci [0, 14] => handler: 17

	at com.generic.BaseTest.initializeWebEnvironment(BaseTest.java:75)
	at com.scripts.desktopGNG.Reception_Entry.loggin(Reception_Entry.java:19)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- loggin -->
</testsuite> <!-- com.scripts.desktopGNG.Reception_Entry -->
